.circleci/config.yml

```diff
@@ -261,6 +261,64 @@ jobs:
             git commit -m "Build documentation for <<pipeline.git.branch>> [ci skip]"
             git push origin gh-pages
 
+  run_benchmarks:
+    docker:
+      - image: cimg/python:<<pipeline.parameters.max_py_ver>>
+    parameters:
+      tmp_results_dir:
+        type: string
+        default: "/tmp/results"
+      checkout_dir:
+        type: string
+        default: "/tmp/checkout"
+      results_dir:
+        type: string
+        default: "/tmp/checkout/ci_benchmark_results"
+      plots_dir:
+        type: string
+        default: "/tmp/checkout/ci_benchmark_results/plots"
+    steps:
+      - install_gpflow:
+          py_ver: <<pipeline.parameters.max_py_ver>>
+          tf_ver: <<pipeline.parameters.max_tf_ver>>
+          tfp_ver: <<pipeline.parameters.max_tfp_ver>>
+      - run:
+          name: Run benchmarks
+          command: |
+            source .venv/bin/activate
+            rm -rf <<parameters.tmp_results_dir>>
+            mkdir -p <<parameters.tmp_results_dir>>
+            python -m benchmark.run --no_plot ci <<parameters.tmp_results_dir>>
+      - add_ssh_keys:
+          fingerprints:  # Add key to give write-access below
+            - "b5:9b:f3:74:26:00:b3:c3:de:3a:b3:03:ee:0b:0b:64"
```

Comment:
Is this fine from a security point of view?

Reply:
> Is this fine from a security point of view?

...I think so... it's not the key itself, only a fingerprint of one...